src/sagemaker/clarify.py

```diff
@@ -278,6 +307,36 @@
                 SchemaOptional("top_k_features"): int,
             },
             SchemaOptional("report"): {"name": str, SchemaOptional("title"): str},
+            SchemaOptional("asymmetric_shapley_value"): {
+                "direction": And(
+                    str,
+                    Use(str.lower),
+                    lambda s: s
+                    in (
+                        "chronological",
+                        "anti_chronological",
+                        "bidirectional",
+                    ),
+                ),
+                "granularity": And(
+                    str,
+                    Use(str.lower),
+                    lambda s: s
+                    in (
+                        "timewise",
+                        "fine_grained",
+                    ),
+                ),
+                SchemaOptional("num_samples"): int,
+                SchemaOptional("baseline"): Or(
```

Comment:
Why is baseline optional? What would be a default value for related_time-series?